### PR TITLE
[PLAY-490] Removed FA from Dist, Imported into Playbook via Yarn

### PR DESCRIPTION
**Pull request by nidaqg**
## Icons rendering as expected
Screen Shot 2022-11-16 at 1 02 11 PM

Screen Shot 2022-11-16 at 1 02 38 PM

#### Breaking Changes

YES. This PR removes Fontawesome from our dist and brings it in directly through yarn. Fonst awesome pro installed in playbook-website, fontawesome free installed in playbook

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-490)

#### How to test this

Test any kits that use icons, such as Icon, Icon Stat Value, Icon Circle, etc. Make sure all icons rendering

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,6 +29,10 @@ RUN apt-get update -y \
 
 WORKDIR /home/app/src
 
+ARG FONT_AWESOME_AUTH_TOKEN=""
```
Comment (benlangfeld) [inline Dockerfile:32]: Better than a build arg would be a secret mount, so that the value doesn't get baked in to the image: https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/reference.md#run---mounttypesecret

File: Jenkinsfile [hunk 2]
```diff
@@ -16,6 +16,20 @@ app.build(
...
+      usernamePassword(
+        credentialsId: 'playbook-ci-build-aws-creds',
+        usernameVariable: 'AWS_ACCESS_KEY_ID',
+        passwordVariable: 'AWS_SECRET_ACCESS_KEY'
+      )
+    ]) {
+      sh "mkdir -p ~/.kube"
+      sh "bin/container/deployer  sops --config .sops.yaml \
+            --decrypt ./config/ci/secrets.yaml \
+            > ./config/ci/secrets.dec.yaml"
```
Comment (benlangfeld) [inline Jenkinsfile:30]:
```suggestion
      sh "bin/container/deployer sops --config .sops.yaml --decrypt ./config/ci/secrets.yaml > ./config/ci/secrets.dec.yaml"
```

**Review by jasperfurniss — APPROVED** [review]

File: playbook/package.json [hunk 3]
```diff
@@ -44,6 +44,7 @@
     "@babel/preset-react": "^7.12.5",
     "@floating-ui/react-dom": "^1.0.0",
     "@floating-ui/react-dom-interactions": "^0.9.1",
+    "@fortawesome/fontawesome-free": "^6.2.1",
```
Comment (thestephenmarshall) [inline playbook/package.json:47]: Why is this added? I would have expected to see:

`"@fortawesome/fontawesome-pro": "^6.2.1"`...

Comment (nidaqg) [reply playbook/package.json:47]: The idea was that the website will have pro installed while playbook as a design system will ship with free as the default. Users have to install the pro version in their codebases themselves if they wish to use it.
